nc_time_axis/tests/unit/test_NetCDFTimeDateLocator.py

```diff
@@ -115,5 +115,43 @@ def test_yearly(self):
             self.check(5, 0, 5*365), [31., 485., 942., 1399., 1856.])
 
 
+class Test_tick_values_yr0(unittest.TestCase):
+    def setUp(self):
+        self.date_unit = 'days since 0001-01-01 00:00'
+
+    def check(self, max_n_ticks, num1, num2, calendar):
+        locator = NetCDFTimeDateLocator(max_n_ticks=max_n_ticks,
+                                        calendar=calendar,
+                                        date_unit=self.date_unit)
+        return locator.tick_values(num1, num2)
+
+    def test_yearly_yr0_remove(self):
+            yr0_remove_calendars = [
+                "proleptic_gregorian",
+                "gregorian",
+                "julian",
+                "standard"
+                    ]
+
+            for calendar in [
+                'standard',
+                'gregorian',
+                'proleptic_gregorian',
+                'noleap',
+                '365_day',
+                '360_day',
+                'julian',
+                'all_leap',
+                '366_day',
+                    ]:
+
+                # convert values to dates, check that none of them has year 0
+                ticks = self.check(5, 0, 100*365, calendar)
+                if calendar in yr0_remove_calendars:
```

Comment:
It's important to also assert that `0` still exists for the other calendars in an `else:` - confirming that you're actually making a difference and it's not just passing my accident.

Reply:
I added another line to address this. Let me know if you think this is sufficient.